lib/cimpress_mcp/client.rb

```
@@ -162,6 +162,24 @@ def get_fulfillment_recommendations(sku:, quantity:, country:, postal_code:)
         return JSON.parse(response)
     end
 
+    def validate_api(swagger_url_to_validate:)
+        RestClient.log=STDOUT
```

probably don't want this, at least by default, maybe hide it behind a verbose arg that default to false?